model/ddl.go

```diff
@@ -107,12 +107,14 @@ func (h *HistoryInfo) Clean() {
 
 // Job is for a DDL operation.
 type Job struct {
-	ID       int64         `json:"id"`
-	Type     ActionType    `json:"type"`
-	SchemaID int64         `json:"schema_id"`
-	TableID  int64         `json:"table_id"`
-	State    JobState      `json:"state"`
-	Error    *terror.Error `json:"err"`
+	ID         int64         `json:"id"`
+	Type       ActionType    `json:"type"`
+	SchemaID   int64         `json:"schema_id"`
+	SchemaName string        `json:"schema_name"`
```

Comment:
Could we not add these fields？
Can we get the schema name and table name from `Query`？